model-archiver/model_archiver/model_packaging.py

```diff
@@ -38,7 +40,9 @@ def package_model(args, manifest):
         # Step 3 : Zip 'em all up
         ModelExportUtils.zip(export_file_path, model_name, model_path, files_to_exclude, manifest)
         logging.info("Successfully exported model %s to file %s", model_name, export_file_path)
-
+    except ModelArchiverError as e:
+        logging.error(e)
```

Comment:
The error message looks like following:
ERROR:root:Given export-path a is not a directory. Point to a valid export-path directory.

Can we configure logging to only print out the message?

Reply:
Sure. We could make that. I don't think this is a blocker for PR though.